driver.php

```diff
@@ -203,17 +203,12 @@
 </p>
 <pre>
 <b>$</b> gcc -v hello.c
-COLLECT_GCC_OPTIONS=...
- /usr/lib/gcc/aarch64-linux-gnu/11/cc1 hello.c ... -o /tmp/ccCKUy4c.s
-COLLECT_GCC_OPTIONS=...
- as -v -EL -mabi=lp64 -o /tmp/ccUkKMFH.o /tmp/ccCKUy4c.s
-COMPILER_PATH=...
-LIBRARY_PATH=...
-COLLECT_GCC_OPTIONS=...
- /usr/lib/gcc/aarch64-linux-gnu/11/collect2 ... /tmp/ccUkKMFH.o ...
+ /usr/lib/gcc/aarch64-linux-gnu/11/cc1 hello.c -o /tmp/ccCKUy4c.s
+ as -o /tmp/ccUkKMFH.o /tmp/ccCKUy4c.s
+ /usr/lib/gcc/aarch64-linux-gnu/11/collect2 /tmp/ccUkKMFH.o
 </pre>
 <p>
-It is a lot harder to read but we can still make out that a compiler step <code>cc1</code> took place, followed by an assembler step <code>as</code>, followed by a linking step via <code>collect2</code>.
+We can still make out that a compiler step <code>cc1</code> took place, followed by an assembler step <code>as</code>, followed by a linking step via <code>collect2</code>.
```

Comment:
I maintain that `gcc` output  is harder to make sense of. You see a "collect" step and it is not obvious what is going on there.